### DISCUSSION TITLE
Welcome to kicad-libraries Discussions!

### DISCUSSION THREAD

**Discussion by pedrominatel** [espressif/kicad-libraries#143] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by al-davis** [comment]
I need a little help.  I'm trying to add EspessIf libraries for schematic and footprint info to KiCad 7.0.
I went to the info page  https://github.com/espressif/kicad-libraries/tree/legacy_kicad7?tab=readme-ov-file
  that kinda explains the process but I'm not able to complete.  Here's what I'm doing :  download the file 
kicad-libraries-legacy_kicad7.zip, Open KiCad, open the Plugin and Content Manager, select libraries, select Install 
from file, select the zip file just downloaded 7 open it  -- then I get a KiCad error box that says "Archive does not
contain a valid metadata.json file"   and all progess stops.
  There is a metadata.json file within the zip that looks coherent.  Did I download the proper zip ?  Did I miss or
misinterpret a step ?    Do something else wrong ?

**Reply by pedrominatel** [reply]
Hi @al-davis,

Can you try [this file](https://github.com/espressif/kicad-libraries/releases/download/2.07/espressif-kicad-addon.zip)?

**Comment by al-davis** [comment]
Yes that works !

Thank you
Al

**Comment by leejsk** [comment]
I would appreciate it if you could read my writing!!!
